gammapy/data/tests/test_observation_summary.py

```diff
@@ -0,0 +1,37 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from astropy.coordinates import SkyCoord
+
+from ...data import DataStore, ObservationTable, ObservationTableSummary
+
+from ...utils.testing import data_manager, requires_data, requires_dependency
+
+@requires_data('gammapy-extra')
+@pytest.fixture
+def summary():    
+    data_store = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
+    target_pos = SkyCoord(83.633083, 22.0145, unit='deg')
+    return ObservationTableSummary(data_store.obs_table, target_pos)
+
+@requires_data('gammapy-extra')
+def test_str():
+    text = str(summary())
+    assert 'Observation summary' in text
+
+@requires_data('gammapy-extra')
+def test_offset():
+    offset = summary().offset
+    assert ((offset.degree.mean() - 1.0) <1.e-3 and (offset.degree.std() - 0.5) <1.e-3)
```

Comment:
This is a bad way to write assertions, because if they fail, you don't get a good error message about what the actual values were. Try it out by putting a wrong value and read the error message!

Please use `assert_quantity_allclose`:
http://docs.gammapy.org/en/latest/development/howto.html#assert-convention